apps/src/block_utils.js

```diff
@@ -1017,10 +1018,17 @@ exports.createJsWrapperBlockCreator = function(
     blockly.Blocks[blockName] = {
       helpUrl: '',
       init: function() {
-        if (color) {
+        // Styles should be used over hard-coded colors in Google Blockly blocks
+        if (style && this.setStyle) {
```

Comment:
Just want to confirm that setStyle is only defined for google blockly blocks?

Reply:
Correct. I added this specifically so that if a CDO Blockly block config had a style (and no color), things wouldn't break.